fastserde/avro-fastserde/build.gradle

```diff
@@ -6,6 +6,40 @@
 
 plugins {
   id "java-library"
+  id "java-test-fixtures"
+}
+
+//Disable fixture publishing
+components.java.withVariantsFromConfiguration(configurations.testFixturesApiElements) { skip() }
+components.java.withVariantsFromConfiguration(configurations.testFixturesRuntimeElements) { skip() }
+
+//This block is only supported and required when building with JDK11
+if (JavaVersion.current() == JavaVersion.VERSION_11) {
+  sourceSets {
+    testFixtures {
+      java {
+        srcDirs = ['src/main/java11']
+      }
+    }
+  }
+
+  compileTestFixturesJava {
+    sourceCompatibility = 11
+    targetCompatibility = 11
+    options.compilerArgs.addAll(['--release', '11'])
+  }
+
+  jar {
+    into('META-INF/versions/11') {
+      from sourceSets.testFixtures.output
```

Comment:
if i run a build with -x test would this break or not include the code in the artifact?